tests/core/usage_statistics/test_usage_statistics.py

```diff
@@ -52,6 +64,141 @@ def in_memory_data_context_config_usage_stats_enabled():
     )
 
 
+def test_comprehensive_list_of_messages():
```

Comment:
A thought after writing a few of these :

1. should `test_comprehensive_list_of_messages()` and associated tests live in a separate file (like `test_schema_validation`) and reserve this file for testing functions that are actually in the `usage_statistics.py` file (like `test_add_datasource_usage_statistics`?)

`test_add_datasource_usage_statistics`